app/views/action_page/index.html.erb

```diff
@@ -10,6 +10,10 @@
       <div class="row">
         <% array_of_3_actions.each do |actionPage|%>
             <div>
+              <% if Category.exists? %>
```

Comment:
`Category` refers to the overall class of categories. 

We want to get the category for current action page (stored in the `actionPage` variable inside this loop). ActiveRecord makes that easy - we can just call `actionPage.category`. 

To check if it's not nil or empty, we do `actionPage.category.present?` I've found this table really helpful for these types of checks: https://stackoverflow.com/a/26877095

Putting it together, you'd have something like
```
<% if actionPage.category.present? %>
  <%= actionPage.category.title %>
<% end %>
```

Reply:
Weird, I swear I tried that and it didn't work. Oh well!